nav-app/src/app/tabs/tabs.component.html

```diff
@@ -48,17 +48,60 @@
     </div> -->
     <div class="newTab border">
         <mat-accordion class="headers-align">
-            <mat-expansion-panel #newlayer hideToggle="true" (click)="newLayer(); newlayer.close()">
+            <mat-expansion-panel #newlayer>
                 <mat-expansion-panel-header>
                     <mat-panel-title>
                         Create New Layer
                     </mat-panel-title>
                     <mat-panel-description>
                         Create a new empty layer
                     </mat-panel-description>
-
                 </mat-expansion-panel-header>
-                <!-- this content will never be shown because clicking opens a new layer tab -->
+
+                <table class="inputTable">                        
+                    <ng-template ngFor let-domain [ngForOf]="filterDomains(ds.versions[0])" let-domainIndex="index">
+                        <td><button mat-button (click)="newLayer(domain.id)">{{domain.name}}</button></td>
+                    </ng-template>
```

Comment:
These buttons need a bit of improvement from a UI perspective:
- They should have a bottom margin to give them space with the inner "more options" panel
- It's hard to tell where the button actually is since they're small and have a lot of horizontal space between them. Making them [raised or stroked](https://material.angular.io/components/button/overview) will help the user know what part of the page is clickable. Also, making them expand to fill the space could also help because the horizontal whitespace is kinda weird. If you do this, consider making them a [button group](https://getbootstrap.com/docs/4.0/components/button-group/) (that link is bootstrap but it shows the UI for button groups).